be/src/column/nullable_column.cpp

```diff
@@ -251,6 +251,24 @@ int NullableColumn::compare_at(size_t left, size_t right, const Column& rhs, int
     }
 }
 
+bool NullableColumn::equals(size_t left, const Column& rhs, size_t right) const {
+    if (immutable_null_column_data()[left]) {
+        return rhs.is_null(right);
```

Comment:
null == null  => null ？

Reply:
Yes, it's right for SQL, but for memory column compute, we should return true?

Reply:
like: map{null: null}

we can't deriver the map's type is MAP<NULL, NULL>, it's will cause so many question, we need modify the type to a primary type, like MAP<BOOLEAN, BOOLEAN>

so, in some functions, like: `cast(map{null: null} as MAP<int, MAP<INT, INT>>)`, we will meet BOOLEAN cast to MAP<INT, INT>